app/src/examples/KeyboardAvoidingView.tsx

```diff
@@ -0,0 +1,196 @@
+    ];
+    const firstNames = ['Emily', 'Alexander', 'Olivia', 'William', 'Sophia'];
+    const lastNames = ['Johnson', 'Lee', 'Hernandez', 'Kim', 'Chen', 'Park'];
+
+    const name = firstNames[number % firstNames.length];
+    const lastName = lastNames[number % lastNames.length];
+    return {
+      text: `I ${likeSynonims[number % likeSynonims.length]} ${
+        hobbies[number % hobbies.length]
+      } and  ${hobbies[(3 * number + 1) % hobbies.length]}`,
+      avatar: `https://i.pravatar.cc/150?img=${number % 7}`,
+      name: `${name} ${lastName}`,
+      username: `${name.toLocaleLowerCase()}${lastName.toLocaleLowerCase()[0]}`,
+    };
+  });
+}
+
+const DATA = getRandomData(20);
+
+const TALL_POPOVER_HEIGHT = 500;
+const SHORT_POPOVER_HEIGHT = 200;
+
+function ListItem({
+  item,
+  onLongPress,
+}: {
+  item: any;
+  onLongPress: (ref: React.RefObject<Animated.View>, height: number) => void;
+}) {
+  const ref = useAnimatedRef<Animated.View>();
+
+  return (
+    <TouchableOpacity
+      style={styles.itemContainer}
+      onPress={() => onLongPress(ref, SHORT_POPOVER_HEIGHT)}
+      onLongPress={() => onLongPress(ref, TALL_POPOVER_HEIGHT)}>
+      <Image source={{ uri: item.avatar + '?u=1' }} style={styles.avatar} />
+      <Animated.View ref={ref} style={styles.item}>
+        <Text style={styles.title}>
+          {item.name} · <Text style={styles.username}>@{item.username}</Text>
+        </Text>
+        <Text>{item.text}</Text>
+      </Animated.View>
+    </TouchableOpacity>
+  );
+}
+
+export default function KeyboardAvoidingViewExample(): React.ReactElement {
+  const [useReanimated, setUseReanimated] = useState(true);
+
+  const [behavior, setBehavior] = useState<
+    'position' | 'height' | 'padding' | undefined
+  >(undefined);
+
+  const KeyboardAvoidingView = useReanimated
+    ? // @ts-ignore
+      Animated.KeyboardAvoidingView
+    : RNKeyboardAvoidingView;
+
+  return (
+    <View style={styles.flexOne}>
+      <View style={styles.header}>
+        <Text>Use Reanimated KeyboardAvoidingView</Text>
+        <Switch
+          value={useReanimated}
+          onChange={(evt) => setUseReanimated(evt.nativeEvent.value)}
+        />
+      </View>
+      <View style={styles.header}>
+        {(['position', 'padding', 'height', undefined] as const).map((key) => {
+          return (
+            <View style={styles.flexOne} key={key || 'undefined'}>
+              <TouchableOpacity
+                style={[
+                  styles.behavior,
+                  behavior === key ? styles.selectedBehavior : undefined,
+                ]}
+                onPress={() => {
+                  setBehavior(key);
+                }}>
+                <Text>{key || 'undefined'}</Text>
+              </TouchableOpacity>
+            </View>
+          );
+        })}
+      </View>
+
+      <KeyboardAvoidingView
+        behavior={behavior}
+        style={styles.flexOne}
+        keyboardVerticalOffset={48 + 10}
```

Comment:
KeyboardVerticalOffset here should be `80` - two views with a height of 40 above the `KeyboardAvoidingView`